### PR TITLE
Add a security policy (Community Standards)

### LINKED ISSUE #47: Add a security policy (Community Standards)

### PR DESCRIPTION
**Pull request by paulo-ferraz-oliveira**
Closes #47.

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+Thanks for helping make nuntius safer for everyone.
+
+Find below updated information on our security policy.
+
+## Security
+
+We take the security of this software seriously.
+
+We don't implement a bug bounty program or bounty rewards, but will work with
+you closed to ensure that your findings gets the appropriate handling.
```
Comment (elbrujohalcon) [inline SECURITY.md:12]: "closed"?

**Review by elbrujohalcon — CHANGES_REQUESTED** [review]

File: SECURITY.md (hunk 1 above)
Comment (paulo-ferraz-oliveira) [reply SECURITY.md:12]: Yeah, it should be "closely". Copy-paste means this error is found elsewhere too 🙄

Comment (paulo-ferraz-oliveira) [reply SECURITY.md]: Removed "closed", changed "findings gets" to "findings get".
Rebased onto the main branch, to resolve merge conflicts, and force-pushed.

Comment (paulo-ferraz-oliveira) [reply SECURITY.md]: a167d0d

**Review by elbrujohalcon — APPROVED** [review]

**Pull request merged**